sourcecode/hub/resources/views/components/mobile-filter-options.blade.php

```diff
@@ -0,0 +1,32 @@
+<div class="offcanvas offcanvas-bottom" id="offcanvasBottomMobile" aria-labelledby="offcanvasBottom" data-bs-backdrop="static" data-bs-keyboard="false">
+    <div class="offcanvas-body">
+        <div class="mb-2">
+            <button
+                id="backButton"
+                class="btn btn-secondary d-lg-none filter-back-button backButton border-0"
+                type="button"
+                data-bs-dismiss="offcanvas"
+                aria-label="{{ trans('messages.back') }}"
+            >
+                <x-icon name="arrow-return-left" class="me-1" />
+            </button>
+        </div>
+
+        <x-filter/>
+
+        <div class="text-center">
+            <button
+                id="showResultsButton"
+                class="btn btn-primary d-lg-none d-md-none filter-button mt-5"
```

Comment:
The `d-lg-none` is unnecessary as the `d-md-none` will hide it for resolutions 'md' and up